plugin/src/test/java/io/jenkins/plugins/casc/ConfigurationAsCodeTest.java

```diff
@@ -54,8 +56,13 @@ public void init_test_from_accepted_sources() throws Exception {
         // should be picked up
         Path target = Paths.get("jenkins.tmp");
         Path newLink = Paths.get(tempFolder.getRoot().getAbsolutePath(), "jenkins_5.yaml");
-        Files.createSymbolicLink(newLink, target);
 
+        try {
+            Files.createSymbolicLink(newLink, target);
+        } catch (Exception e) {
```

Comment:
could we just catch IOException instead?
```suggestion
        } catch (IOException e) {
```

Reply:
Yeah that would make sense since we already know the type of exception :+1: